isg-503.coffee

```diff
@@ -21,9 +34,9 @@ module.exports =
 	yocto:
 		machine: 'isg-503'
 		image: 'balena-image-flasher'
```

Comment:
HI @RockSupport , can you please confirm if this device image a flasher one? Does it have an sd-card slot which is used to load the flasher image for provisioning, so that it can write the balenaOS image to the emmc?

If yes, can you please check the instructions that get generated in isg-503.json when triggering the yocto build of this branch with:
`balena-yocto-scripts/build/barys -b build_isg2 -m isg-503 -d`

Please let us know if the instructions need to be updated to mention a switch that might need to be toggled to boot from SD or eMMC.

Reply:
There is an sd-card slot in isg-503 ，support boot and update from sd card

Reply:
@RockSupport does the device also boot from eMMC? If yes, which button/switch needs to be switch to toggle between eMMC and SD-CARD?

Reply:
Yes，the device can boot from eMMC。But it boot from sd card first，if sd-card has boot partition